applications/app/pages/ContentHtmlPage.scala

```diff
@@ -33,12 +33,25 @@ object ContentHtmlPage extends HtmlPage[Page] {
   def html(page: Page)(implicit request: RequestHeader, applicationContext: ApplicationContext): Html = {
     implicit val p: Page = page
 
+    def getMedia(page: MediaPage): Html  = {
+
+      if(ActiveExperiments.isParticipating(AudioPageChange)) {
+
+        page.media match {
+          case a: Audio => audioBody(page, displayCaption = false)
+          case _ => mediaBody(page, displayCaption = false)
+        }
+      } else {
+        mediaBody(page, displayCaption = false)
+      }
+    }
+
     val bodyClasses: Map[String, Boolean] = defaultBodyClasses() ++ Map(
       ("is-immersive", Page.getContent(page).exists(_.content.isImmersive))
     )
     val content: Html = page match {
       case p: ImageContentPage => imageContentBody(p)
-      case p: MediaPage => mediaBody(p, displayCaption = false)
+      case p: MediaPage => getMedia(p)
```

Comment:
For consistency, you may consider naming the method `audioOrMediaBody` rather than `getMedia`.